src/events/tab_event_handler.ts

```diff
@@ -43,8 +43,10 @@ export class tabEventHandler {
 		if (!repoId) return
 		// Get list of current tabs
 		const openTabs = vscode.window.tabGroups.all;
-		const tabs: ITabFile[] = openTabs.flatMap(tabGroup => 
+	try {	
+			const tabs: ITabFile[] = openTabs.flatMap(tabGroup => 
 			tabGroup.tabs.map(tab => {
+				if (!tab.input) return null;
 				// Get path of tab
 				// @ts-ignore
 				const tabFilePath = tab.input.uri.path;
```

Comment:
Any possibility `uri` can be undefined here?

Reply:
We can update this as:
			`if (!tab.input | !tab.input?.uri) return null;`

Reply:
But the error says ```reading "path"```,  why is that? @saadberry 
We didn't observe the error of reading "uri", did we?

Reply:
We did not. Maybe in that specific case, the `path` value was not defined. 

Regardless, I believe our added checks should prevent that error from occurirng 
> We didn't observe the error of reading "uri", did we?

Reply:
ok